test_directo.obj : unable to open file
